.github/actions/test/action.yml

```
@@ -54,11 +55,11 @@ runs:
           --tmpfs /tmp_dir_test:exec \
           ${{inputs.image}} \
           bash /src/run.sh
-        if docker cp "$(cat docker.cid)":/tmp/tests.xml tests.xml
+        if docker cp "$(cat docker.cid)":/tmp/output_tests ./
```

```suggestion
        if docker cp "$(cat docker.cid)":${CTEST_DIR} ./
```
Can you not use the variable here ? I tested this on my machine and it works with a variable.

---

The container does not have access to the shell variables. Also, outside the scope of this PR.